tests/behavioural/features/steps/status_code.py

```diff
@@ -2,31 +2,31 @@
 from behave import then
 
 
-@then('a success status code (200) is returned')
+@then('a success status code 200 is returned')
```

Comment:
I know I'm being picky, but this could be flattened into a single function that reads:
```@then('a status code {status_code} is returned')``` and save having to have a whole module for it.  
If you don't want to do it here, can you create a github issue/trello card so that it gets done please?

Reply:
Definitely worth doing, but I'd suggest a different task/card, especially as this is quite a big pr anyhow

Reply:
Card created 👍